### PR TITLE
Reweighting

### PR DESCRIPTION
**Pull request by urbach**
some more improvements

### CODE REVIEW

**Comment by kostrzewa** [comment]
In order to study the error scaling with the number of reweighting samples without having to rerun the full computation each time, I think it would be useful to extend the output further and either provide seperate files for each sample or a sample index in the output file. This of course also means extending the ```sum[j]``` and ```sum_sq[j]``` arrays with another index.

File: reweighting_factor.c [hunk 1]
```diff
@@ -83,13 +83,15 @@ void reweighting_factor(const int N, const int nstore) {
   }
   
   if(g_proc_id == 0) {
-    ofs = fopen("reweighting_factor.data", "a");
-    fprintf(ofs, "%d ", nstore);
+    char filename[50];
+    sprintf(filename, "reweighting_factor.data.%.5d", nstore);
+    ofs = fopen(filename, "w");
```
Comment (kostrzewa) [inline reweighting_factor.c]: file open check?

**Comment by urbach** [comment]
ah, the trlog part is still missing, isn't it?

**Comment by kostrzewa** [comment]
True, with different kappas is doesn't cancel...

**Comment by urbach** [comment]
@kostrzewa could you crosscheck whether I got the signs correct?

**Comment by kostrzewa** [comment]
Thanks, as discussed "offline", this looks fine now.

**Pull request merged**